keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -0,0 +1,705 @@
+
+*   **Name** - a well known or custom name to identify the claim.
+*   **Value** - a claim-dependent string.
+
+
+### Well known claims
+
+The requirements below use the keywords **must, should,** and **may** purposefully in accordance with [RFC-2119](https://tools.ietf.org/html/rfc2119).
+
+
+#### Claim: `id.k8s.io`
+
+Contains a unique identifier for the containing cluster.
+
+
+##### Uniqueness
+
+*   The identifier **must** be unique within the ClusterSet to which its cluster belongs for the duration of the cluster’s membership.
+*   The identifier **should** be unique beyond the ClusterSet within the scope of expected use.
+*   The identifier **may** be globally unique beyond the scope of its ClusterSet.
+*   The identifier **may** be unique beyond the span of its cluster’s membership and lifetime.
+
+
+##### Lifespan
+
+*   The identifier **must** exist and be immutable for the duration of a cluster’s membership in a ClusterSet, and as long as a `clusterset.k8s.io` claim exists.
+*   The identifier **must** exist for the lifespan of a cluster.
+*   The identifier **may** be immutable for the lifespan of a cluster.
+
+
+##### Contents
+
+*   The identifier **must** be a valid RFC-1123 DNS label.
+    *   Following the most restrictive standard naming constraint ensures maximum usefulness and portability.
+    *   Can be used as a component in MCS DNS.
+*   The identifier **may** be a human readable description of its cluster.
+
+
+##### Consumers
+
+*   Can rely on the identifier existing, unmodified for the entire duration of its membership in a ClusterSet.
+*   **Should** watch the `id.k8s.io` claim to handle potential changes if they live beyond the ClusterSet membership.
+*   **May** rely on the existence of an identifier for clusters that do not belong to a ClusterSet so long as the implementation provides one.
+
+
+#### Claim: `clusterset.k8s.io`
+
+Contains an identifier that relates the containing cluster to the ClusterSet in which it belongs.
+
+
+##### Lifespan
+
+*   The identifier **must** exist and be immutable for the duration of a cluster’s membership in a ClusterSet.
+*   The identifier **must not** exist when the cluster is not a member of a ClusterSet.
+
+
+##### Contents
+
+*   The identifier **must** associate the cluster with a ClusterSet.
+*   The identifier **may** be either unique or shared by members of a ClusterSet.
+
+
+##### Consumers
+
+*   Can rely on the identifier existing, unmodified for the entire duration of its membership in a ClusterSet.
+*   **Should** watch the clusterset claim to detect the span of a cluster’s membership in a ClusterSet.
+
+
+### Additional Claims
+
+Implementers are free to add additional claims as they see fit, so long as they do not conflict with the well known claims. `k8s.io` and `sigs.k8s.io` claims are reserved for Kubernetes and related projects.
```

Comment:
How about something else that matches `*.k8s.io`

Reply:
Restricted `*.k8s.io` and `*.kubernetes.io` from additional claims in e5026d65e970576d5e8ebb648fe05671b741129f